services/web/client/source/class/qxapp/component/widget/inputs/NodeOutputLabel.js

```diff
@@ -88,14 +92,16 @@ qx.Class.define("qxapp.component.widget.inputs.NodeOutputLabel", {
   },
 
   members: {
+    __portId: null,
```

Comment:
prop?

Reply:
I am using it only internally so I think for the moment there's no need to generate any accessors/setters